packages/popover/src/Popover/Popover.stories.jsx

```diff
@@ -212,8 +214,17 @@ const InnerContent = ({ handleClose }) => (
   </Box>
 )
 
-export const PassesThemeToContent = () => {
-  return (
+export const PassesThemeToContent = {
+  play: async ({ args, canvasElement }) => {
+    const canvas = within(canvasElement)
+    const text = canvas.getByText('hello world')
+    expect(text).toBeInTheDocument()
+    await userEvent.hover(text)
```

Comment:
Shouldn't this be a click event?

Reply:
Hmm, seems like the `openOnHover` is set to true as part of the args: 

https://github.com/priceline/design-system/pull/1348/files#diff-d81c37b8b7a054bb7e0a0ed0441ad294a5fe202bd3b9667df8750d76647c5001R21